src/lib/src/ng-select.component.ts

```diff
@@ -134,6 +134,11 @@ export class NgSelectComponent implements OnInit, OnDestroy, ControlValueAccesso
                 case KeyCode.Esc:
                     this.close();
                     break;
+                case KeyCode.Backspace:
+                    if (this.multiple) {
+                        this.unSelectLastItem();
+                    }
```

Comment:
Would be better to create method called `handleBackspace` and put what logic there. Some logic for single select is missing. It can be:
```js
private handleBackspace() {
   if (this.multiple) {
      this.unSelectLastItem();
   } else {
      // Single select logic
      // 1) If input is focused and there is some value selected by clicking backspace I expect to clear model, the same as clicking on crisis icon. This way we can ensure better accessibility because now it is possible to clear selected value only on clear icon click
      // 2) In case if some filter text is inserted by clicking backspace should work as default behaviour for deleting filter text
   }
}
```